infrastructure/server/oidc_clients_spar.tf

```diff
@@ -0,0 +1,105 @@
+resource "aws_cognito_user_pool_client" "dev_spar_oidc_client" {
+  access_token_validity                         = "5"
+  allowed_oauth_flows                           = ["code"]
+  allowed_oauth_flows_user_pool_client          = "true"
+  allowed_oauth_scopes                          = ["openid", "profile", "email"]
+  callback_urls                                 = [
+    "https://oidcdebuggersecure-3d5c3f-dev.apps.silver.devops.gov.bc.ca/",
+    "http://localhost:3000/dashboard",
+    "http://localhost:3000/silent-check-sso"
+  ]
+  logout_urls                                   = [
+    "https://oidcdebuggersecure-3d5c3f-dev.apps.silver.devops.gov.bc.ca/",
```

Comment:
Don't forget to add the logout chain!! Fix this in another pull request https://github.com/bcgov/nr-forests-access-management/pull/495